manifests/t/tawk/to/2.2.0/tawk.to.installer.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: tawk.to
+PackageVersion: 2.2.0
+Installers:
+- InstallerUrl: https://windows.tawk.to/v2/releases/tawk.to-latest.exe
+  Architecture: x64
+  InstallerType: nullsoft
+  InstallerSha256: C602D7A29AC2533904AF0B91801CF58DE698C73E7D211F439522B66554AE9DFC

```

```suggestion
- InstallerUrl: https://windows.tawk.to/v2/releases/tawk.to-latest.exe
  Architecture: x64
  InstallerType: nullsoft
  InstallerSha256: C602D7A29AC2533904AF0B91801CF58DE698C73E7D211F439522B66554AE9DFC
- InstallerUrl: https://windows.tawk.to/v2/releases/tawk.to-latest.exe
  Architecture: x86
  InstallerType: nullsoft
  InstallerSha256: C602D7A29AC2533904AF0B91801CF58DE698C73E7D211F439522B66554AE9DFC
```
Website says the package is valid for x86 and x64 systems